packages/core/src/addresses/redux/actions/doDeleteDefaultShippingAddress.js

```diff
@@ -0,0 +1,53 @@
+import * as actionTypes from '../actionTypes.js';
+import { getDefaultAddress } from '../reducer.js';
+
+/**
+ * Responsible for deleting the users default Shipping address.
+ *
+ * @param deleteDefaultShippingAddress - Delete default Shipping address client.
+ *
+ * @returns Thunk factory.
+ */
+const removeDefaultShippingAddressFactory =
```

Comment:
```suggestion
const doDeleteDefaultShippingAddressFactory =
```

Reply:
The const was removed to fit the pattern followed by the other actions.

Reply:
Thanks!